mmdet/datasets/pipelines/transforms.py

```diff
@@ -872,5 +872,5 @@ def __call__(self, results):
 
     def __repr__(self):
         repr_str = self.__class__.__name__
-        repr_str += '(transformations={})'.format(self.transformations)
+        repr_str += '(transformations={})'.format(self.transforms)
```

Comment:
Thanks! You may also change the key here from `transformations` to `transforms`.

Reply:
Thanks for the comment. Updated PR.